src/File.py

```diff
@@ -0,0 +1,77 @@
+import os, time
+
+class File:
+    def __init__(self, path):
+        self.name = path.split('/')[-1] if path != '/' else '/'
+        self.path = path
+        self.is_file = os.path.isfile(path)
+        self.size, self.parent_size = 0, 0
+        self.next_files = []
+
+    def __repr__(self):
+        return "File {}".format(self.name)
+
+    def get_file_info(self):
+        files_count, count_subdirs = self.count_files_and_subdirs()
+        return {
+            "name": self.get_name(),
+            "path": self.get_path(),
+            "type": self.if_is_file(),
```

Comment:
I've said it previously that "type" is not appropriate name :p

Reply:
Okay, I will change it to "is_file".